.github/dependabot.yml

```diff
@@ -26,16 +27,17 @@ updates:
         update-types:
           - "minor"
           - "patch"
-  - package-ecosystem: "docker"
-    directory: "/images"
+  - package-ecosystem: "gomod"
+    directories:
+      - "/"
+      - "**/rootfs"
     schedule:
       interval: "weekly"
     labels:
       - "area/dependency"
       - "release-note-none"
       - "ok-to-test"
     groups:
-      actions:
+      all:
```

Comment:
This is the name that will be included in the PR, so we can call this `gomod` and update the others as well to use meaningful names

Reply:
Yeah, sure, sounds good!